multisig-ism-metadata/src/main.sw

```diff
@@ -0,0 +1,72 @@
+library metadata;
+
```

Comment:
Wondering if we should link to the Solidity impls of this and multisig-ism as a reference